wordpress/wp-content/plugins/sil-dictionary-webonary/include/configuration.php

```diff
@@ -296,6 +296,41 @@ function webonary_conf_widget($showTitle = false)
 	if(isset($_POST['btnSaveRelevance']))
 		relevanceSave();
 
+	$import_status = '';
```

Comment:
This line isn't needed. The value of `$import_status` is overwritten in both the `if` and `else` blocks below

Reply:
Thanks. Been doing too much Typescript lately...